tests/js/user-tests.js

```diff
@@ -150,6 +150,21 @@ module.exports = {
     TestCase.assertEqual(err.message, "function not found: 'error'");
   },
 
+  async testRemoteMongoClient() {
+    let app = new Realm.App(appConfig);
+    let credentials = Realm.Credentials.anonymous();
+    let user = await app.logIn(credentials);
+
+    let mongo = user.remoteMongoClient('BackingDB');
+    let collection = mongo.db('test').collection('testRemoteMongoClient');
+
+    await collection.deleteMany({});
+    await collection.insertOne({hello: "world"});
```

Comment:
Is this relying on a cleanup defined elsewhere or does this require a manual reset of the database?

Reply:
Line 161 does cleanup so the rest of the test always runs in a clean state.

Reply:
🤦 Never mind ...